### ISSUE TITLE
Merge paths help

### ISSUE THREAD

**Issue by kerrylofton** [airbnb/lottie-web#675]
I am working on a huge interactive project with several attached scenes and layers all converted to shape layers from AI. After going through and deleting all of the merge paths from the contents, not all will go nicely. When I delete it from these tree layers it completely changes the look and I can't figure out why or how. Like I said, all the other layers allowed me to delete the merge paths with no issues. Any thoughts on how I can get around this? I saw in another issue about recreating the shapes with mask but Im not sure if that's possible with the design of these tree. I am attaching screenshots of how they look with and without the merge paths.

![screen shot 2017-10-14 at 2 27 47 pm](https://user-images.githubusercontent.com/32747720/31578691-2d0cbae4-b0ec-11e7-8024-26f0ef86a7cf.png)
![screen shot 2017-10-14 at 2 28 05 pm](https://user-images.githubusercontent.com/32747720/31578692-2d1fd1f6-b0ec-11e7-9148-68bf12e097c2.png)

**Comment by bodymovin** [comment]
Can you share the .aep?

**Comment by kerrylofton** [comment]
Im not able to due to the confidentiality of this project however I may have found a work around with the steps below. Do you see any issues with this process?

Create a new shape layer
auto trace the original tree layer
copy the mask path 
paste the mask path on the new created shape layer


That seems to work so far. If I leave the original tree layer hidden in the comp with the merge paths (for future reference or if something needs to be changed) will that affect the export from bodymovin? Or should I just delete the original layers that still have the merge paths?

**Comment by bodymovin** [comment]
Make sure the shapes you use are not too complex, or it will affect performance.
If you can, try using the canvas renderer and precompose shapes that you're not animating.

**Comment by kerrylofton** [comment]
So I went through and replaced all the merge paths layers with new shape layers but there were some animations that I didn't want to recreate so I just hid the layer that had the merge paths but parented the new shape layer to it. After exporting I noticed some weird things like things exporting as just solid colors. Do I have to delete those layers with the merge paths entirely? Or is it the fact that I am parenting to the merge paths? Does parenting work or is that an issue within the program as well? Sorry for all the questions, this is a massive interactive project with several connected scenes so every time we jump one hurdle we find many more haha
